sklearn/model_selection/tests/test_search.py

```diff
@@ -254,6 +255,15 @@ def test_grid_search_groups():
         gs.fit(X, y)
 
 
+def test_grid_search_classes_parameter():
+    # Verify that the GridSearchCV can pass through the classes_ attribute
+    clf = MockClassifier()
+    grid_search = GridSearchCV(clf, {'foo_param': [1, 2, 3]})
+
+    grid_search.fit(X, y)
+    assert_array_equal(grid_search.classes_, np.unique(y))
```

Comment:
Please add a test to check that `hasattr(gscv, 'classes_')` is false given a regressor?